changelog/645.misc.rst

```diff
@@ -0,0 +1 @@
+Fix the grammatical error in ``Guild`` channel prepoerties.
```

Comment:
```suggestion
Fix the grammatical error in :class:``Guild`` channel properties.
```
This is also a docs change, `645.doc.rst` would be more suitable

Reply:
https://github.com/DisnakeDev/disnake/pull/645/commits/fe8096b8a341094e0b4793e92f468efa07dfa04e